packages/select/src/utils.tsx

```diff
@@ -8,6 +8,10 @@ import {
 } from './Option';
 import { InternalOptionGroup, OptionGroupElement } from './OptionGroup';
 import { isFragment } from 'react-is';
+import { breakpoints } from '@leafygreen-ui/tokens';
+
+// Any screen smaller than a tablet with a coarse pointer and no hover capability (i.e. touch screen)
```

Comment:
Maybe you can explain a little what coarse pointer means? 

A lot of docs say `pointing device of limited accuracy` but thats sorta hard to comprehend without any additional context.  I interpreted it as, you can press down on a touchscreen but your press may not always be accurate where as with a mouse it should be accurate because the pointer is smaller(fine).